php/examples/VaasExample/GetVerdictByFile.php

```diff
@@ -13,9 +13,9 @@
     getenv("TOKEN_URL") ?: "https://account.gdata.de/realms/vaas-production/protocol/openid-connect/token"
 );
 
-$vaas = new Vaas(
-    getenv("VAAS_URL") ?? "wss://gateway.production.vaas.gdatasecurity.de"
-);
+$vaas = (new Vaas())
+    ->WithUrl(getenv("VAAS_URL") ?? "wss://gateway.production.vaas.gdatasecurity.de")
+    ->build();
 
 $vaas->Connect($authenticator->getToken());
```

Comment:
Just to be clear. With the new implementation the Connect is not necessary anymore. Maybe I sould also give an example on how to use it without you having to call connect yourself.

Reply:
If fixed that in my new commit.